### ISSUE TITLE
[Bug]: Obersverbility warning in windows with bal test command

### ISSUE THREAD

**Issue by anupama-pathirage** [ballerina-platform/ballerina-lang#42211]
### Description

When using the `bal test` command, you can see the following warning in windows. Bal run and build commands are working fine.

```
Compiling source
        dumin/hello_world:0.1.0
        warning: Failed to add Observability information to Jar due to: C:\anupama\ballerinatest\hello_world\target\cache\dumin\hello_world\0.1.0\observe\dumin-hello_world-observability-symbols.jar: The process cannot access the file because it is being used by another process

Running Tests

        hello_world


                1 passing
                0 failing
                0 skipped
```

### Steps to Reproduce

Try the steps in https://ballerina.io/learn/test-ballerina-code/test-a-simple-function/

Then issue the `bal test` command

### Affected Version(s)
Windows 11

Ballerina 2201.8.4 (Swan Lake Update 8)
Language specification 2023R1
Update Tool 1.4.2

### Related area

-> Compilation

**Comment by azinneera** [comment]
The jar will be removed with https://github.com/ballerina-platform/ballerina-lang/pull/42002. So the issue will go away after the fix.

**Comment by NipunaMadhushan** [comment]
The jar has been removed with #42002. This will be included with Update 9 release.

**Comment by anupama-pathirage** [comment]
@azinneera / @NipunaMadhushan What is the possiblity of adding this to next patch release?

**Comment by NipunaMadhushan** [comment]
@anupama-pathirage Since we have removed generating a jar, In my opinion, it's best to go with a new update release (Update 9). I will check with @warunalakshitha and @keizer619 whether we can remove this from 2201.8.x as well.

**Comment by NipunaMadhushan** [comment]
@anupama-pathirage I will send the PR to remove generating symbols.jar in `2201.8.x`. It will be included in the `2201.8.6` release.

**Comment by NipunaMadhushan** [comment]
Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/42252
